pkg/spdk/engine.go

```
@@ -583,13 +664,42 @@
+			return false
+		}
+
+	}
+	return true
+}
+
 func (e *Engine) ValidateAndUpdate(spdkClient *spdkclient.Client) (err error) {
 	updateRequired := false
 
+	existingNvemSubsystem := api.NvmeSubsystem{}
+	copier.Copy(&existingNvemSubsystem, &e.NvmeSubsystem)
```

golangci-lint (1.62.2)

Error return value of `copier.Copy` is not checked

(errcheck)